portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,36 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Emma's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>My(Emma's) Portfolio</h1>
+      <h2>About Me</h2>
+      <p>
+          My name is Emma Wenzel; I'm a STEP intern with Google for summer 2020. 
+          Starting in August, I'll be a sophomore at the University of Colorado
+          Boulder. I like acting and rock climbing, and I'm currently watching 
+          Avatar for the first time. 
+      </p>
+        <a href="https://github.com/EmmaWenzel">GitHub link</a>
+      <h5>
+           This is a picture of a seagull. Seagulls are my second favorite
+           type of bird. This particular seagull is french; we'll call him
+           Louis.
+      </h5>
+      <img src="/images/IMG_5916.JPG" alt="seagull?" width = "300">
+
+      <p>This is my portfolio. Doesn't it look nice?</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
+      <p>Click here to get a random fact:</p>
+      <button onclick="addRandomFact()">facts.</button>
```

Comment:
not a required change, but your button will say "facts." inside of it. It might be helpful to have something like "Get a fact!" inside there instead.

Reply:
I changed it to Facts!